### PR TITLE
F#54 oms remove pun

### PR DESCRIPTION
**Pull request by sweetclimusic**
There are quite a lot of Photon Pun things removed. Merging just these changes before adding more bolt code.

### CODE REVIEW

File: Assets/Scripts/MainMenu/MainMenuButtons.cs [hunk 1]
```diff
@@ -15,46 +15,47 @@ public class MainMenuButtons : MonoBehaviour
     public Toggle sfxToggle, musicToggle, overallToggle;
     public Slider sfxSlider, musicSlider, overallSlider;
     public int ScenePosition = 3;
-    
+    [SerializeField]
```
Comment (massieb) [inline Assets/Scripts/MainMenu/MainMenuButtons.cs]: This optimization is already in a separate branch

Comment (massieb) [reply Assets/Scripts/MainMenu/MainMenuButtons.cs]: Remove it in this branch

Comment (sweetclimusic) [reply Assets/Scripts/MainMenu/MainMenuButtons.cs]: I don't understand how this was in the branch? I reverted this file see commit d268625

Comment (sweetclimusic) [reply Assets/Scripts/MainMenu/MainMenuButtons.cs]: Your comment on that version of the file is also flagged as Outdated on github. The PR does have that removed in favor of the F#61 PR.

Comment (Dejin1) [reply Assets/Scripts/MainMenu/MainMenuButtons.cs]: If its taken out it should be fine. Though I would suggest we wait to see what Marcel says as he brought up the comment and I am not entirely adept in this matter.

Comment (sweetclimusic) [reply Assets/Scripts/MainMenu/MainMenuButtons.cs]: @Dejin1 yeah, it is removed. This line of comments is on a old commit and not the current being used to do PR.

**Pull request merged**